examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -6,7 +6,7 @@ import {
   stringToBytes,
   TransactionType
 } from '@algorand-builder/algob';
-import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js/build/src/index';
+import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js';
 import { assert } from 'chai';
 
 import { getAcc } from './common';
```

Comment:
I noticed we're using `import {..}` here in `.js`, whereas in other files we are using `const {..} = require('..');`. Is there any specific reason for doing that?

Reply:
no, not any specific reason